source/reference/permissions.txt

```diff
@@ -0,0 +1,27 @@
+.. _c2c-permissions-and-roles:
+
+================
+User Permissions
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/fact-permissions-body.rst
+
+
+Self-Managed Clusters
+---------------------
+
```

Comment:
**Suggestion**

Add a small blurb after the title giving more detail on the differences between the two tables.

This could be a simple reminder that self-managed clusters have the extra permission requirement.

Ex: Self-Managed Clusters

Self-Managed Clusters require an additional cluster monitoring permission for each sync type.